docs/pages/components/dropdown/Dropdown.vue

```diff
@@ -2,8 +2,10 @@
     <div>
         <Example :component="ExSimple" :code="ExSimpleCode">
             <p>
-                While it appear as a modal for tablet and smartphones,
-                Dropdowns with <code>hoverable</code> prop won't change it's behavior to avoid any malfunction with hover.
+                It appears as a modal for tablets and smartphones, more precisely, when the screen is narrower than <code>$dropdown-mobile-breakpoint</code>.
+                However, Dropdowns with <code>"hover"</code> trigger won't change it's behavior to avoid any mulfunction with hover.
```

Comment:
Below are some minor grammatical changes that you can copy/paste👇🏼.
```
It appears as a modal for tablets and smartphones, more precisely when the screen is narrower than <code>$dropdown-mobile-breakpoint</code>. However, dropdowns with <code>"hover"</code> trigger won't change its behavior to avoid any malfunction with hover. <code>"hover"</code> trigger works like <code>"click"</code> trigger on touch devices where hover events do not make sense.<code>"hover"</code> trigger precedes <code>"click"</code> trigger.
```